packages/frontend/src/components/Terminal/Terminal.module.css

```diff
@@ -1,4 +1,8 @@
-.container,
+.container {
+  width: 100%;
+  height: 100%;
+  background-color: black;
```

Comment:
```suggestion
  
```

Reply:
remove?

Reply:
Terminal does not always fill the container since it always adapts to integers of a terminal row height.

This line is the whole purpose of this PR.

Reply:
Well, you are adding 3 lines, and the first 2 seem to sovle the problem.
I can not replicate any difference in behaviour toggling the backround color line.